### ISSUE TITLE
Unable to Edit the Amendments

### ISSUE THREAD

**Issue by FaiqaShafqat79** [LibreHealthIO/lh-ehr#1417]
**A. My Outreachy username :** faiqa 

**B. Issue Title :** Unable to Edit the Amendments

**C. Bug Report Date :** March 5, 2019 

**D. Site Affected :** Documentation Site. 

**E. OS/Browser Used :** Windows/Chrome. 

**F. Workflow Module/Tab :** Patient Summary. 

**G. Step to reproduce :** 

1. Login with your account credentials. 
2. Take your cursor to the Patient/Client. It gives the dropdown menu. 
3. Click the Finder. It opens the Patient Finder tab.
4. Click on any patient from the given patients list. It will open the Patient Summary tab.
5. Take cursor to the Amendments and click on Edit.

**H. Expected Behavior :** To be able to make changes when we choose the Edit option.

**I. What Actually Happened :** When clicked on one of the amendments it opened the details of that amendment which were in read only state and couldn’t be edited.

**J. ScreenShots :** 
1. Click on the Edit button.
![amendment-v1](https://user-images.githubusercontent.com/37265494/53834217-440ae400-3fac-11e9-9e44-84b5a90196bf.png)

2. Click on the Amendment that you want to edit.
![amendment-v2](https://user-images.githubusercontent.com/37265494/53834290-70befb80-3fac-11e9-88eb-c4109efd42aa.png)

3. It only displays the data. You can’t edit it.
![amendment-v3](https://user-images.githubusercontent.com/37265494/53834335-8d5b3380-3fac-11e9-898f-cb286687babc.png)

**H). Estimation of the bug severity :** Module's function ceases at this point.

**Comment by muarachmann** [comment]
@konikodes are we required to edit those., @FaiqaShafqat79 did you very this with Konikodes? that it actually needs to be edited?

**Comment by FaiqaShafqat79** [comment]
@muarachmann Yes, I discussed this issue with @KoniKodes first but didn't discussed that it actually needs to be edited

**Comment by KoniKodes** [comment]
@FaiqaShafqat79 What happens if you try to add it first?

**Comment by muarachmann** [comment]
Please assert with Toni if this is necessary thanks

**Comment by FaiqaShafqat79** [comment]
> @FaiqaShafqat79 What happens if you try to add it first?

when I add it first it gives me this screen


![Amendments add v1](https://user-images.githubusercontent.com/37265494/54077327-84ce6a00-42d8-11e9-8be7-7ca57c2c1510.png)

on save
![Amendment-save v1](https://user-images.githubusercontent.com/37265494/54077334-9c0d5780-42d8-11e9-91ef-e9401adda718.png)

**Comment by KoniKodes** [comment]
Did the added testing show up on the new List?

**Comment by FaiqaShafqat79** [comment]
No. when I click on add it gives me this screen
![Amendments add v2](https://user-images.githubusercontent.com/37265494/54096658-8712f000-43ce-11e9-94b0-0ce607390d37.png)

If I filled the fields and save, It gives my a new list
![Amendment-save v1](https://user-images.githubusercontent.com/37265494/54096734-e2dd7900-43ce-11e9-8965-f311c49522d3.png)



The problem is there is an edit button on amendemnts that should allow us to edit. but there is no option to edit
![Amendment-v4](https://user-images.githubusercontent.com/37265494/54096973-0228d600-43d0-11e9-918e-8e5eb47ac578.png)


I think it should be something like disclousre
![disclouser v1](https://user-images.githubusercontent.com/37265494/54097261-0f929000-43d1-11e9-9b42-4aa91b7b3603.png)


when we click on edit button it gives us this screen
![disclouser v2](https://user-images.githubusercontent.com/37265494/54097425-deff2600-43d1-11e9-949c-6fac56bfbf80.png)

Hope you got my point. Kindly guide me am I taking this right ?

**Comment by KoniKodes** [comment]
I need to check to see if this is an authorized user issue.
